spaces/domains.json

```
@@ -229,5 +229,5 @@
   "vote.flashstake.io": "flashcommunity.eth",
   "vote.fenixtoken.net": "fenixtoken.eth",
   "vote.liquifi.org": "liquifi.eth",
-  "vote.x0swap.com": "x0swap.eth"
+  "vote.honeydao.com": "honeydao.eth",
```

Please remove the comma from the last line.
```suggestion
  "vote.honeydao.com": "honeydao.eth"
```